### PR TITLE
Remove sorting of ts.df by timestamps in plot methods

### LINKED ISSUE #389: Remove sorting of `ts.df` by timestamps in plot methods
### 🚀 Feature Request

There are a few places there we for some reason sort `ts.df` by timestamp:
- [`plot_forecast`](https://github.com/etna-team/etna/blob/b011ec8fae0a9711fe9fa16a1e9e2227749f6e8f/etna/analysis/forecast/plots.py#L127)
- [`plot_forecast_decomposition`](https://github.com/etna-team/etna/blob/b011ec8fae0a9711fe9fa16a1e9e2227749f6e8f/etna/analysis/forecast/plots.py#L917)

It doesn't make much sense because it shouldn't be possible to have a wrong order here due to using `freq` in `TSDataset`.

### Proposal

Remove this sorting or understand why do we need it. Try to find other such places and fix them too.

### Test cases

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by kenshi777**
## Before submitting (must do checklist)

- [ ] Did you read the [contribution guide](https://github.com/tinkoff-ai/etna/blob/master/CONTRIBUTING.md)?
- [ ] Did you update the docs? We use Numpy format for all the methods and classes.
- [ ] Did you write any new necessary tests?
- [ ] Did you update the [CHANGELOG](https://github.com/tinkoff-ai/etna/blob/master/CHANGELOG.md)?

## Closing issues

closes #389

### CODE REVIEW

**Comment by Ama16** [comment]
There are exist 2 another `sort_values`, use ctrl+F to find

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Fix rendering in 210 tutorial ([#386](https://github.com/etna-team/etna/pull/386))
-- 
+- Remove sorting of `ts.df` ([#389](https://github.com/etna-team/etna/issues/389))
```
Comment (d-a-bunin) [inline CHANGELOG.md:32]: - This should contain a link to PR, not to issue
- This should give more details
```
Remove sorting of `ts.df` by timestamps in `plot_forecast` and `plot_forecast_decomposition` ([#410](https://github.com/etna-team/etna/pull/410))
```

**Review by d-a-bunin — CHANGES_REQUESTED** [review]
Look at comments above.

File: CHANGELOG.md (hunk 1 above)
Comment (kenshi777) [reply CHANGELOG.md]: fixed it

**Review by d-a-bunin — APPROVED** [review]

**Pull request merged**